src/components/receipt-view/data-section/DataSection.js

```diff
@@ -9,51 +9,108 @@ import ImpactDonutAggregate from './ImpactDonutAggregate';
 // Styles
 import styles from '../styles/data-section.module.css';
 
-const DataSection = () => (
+const LAND = 'land';
+const WATER = 'water';
+const EUTRO = 'eutro';
+const GHG = 'ghg';
```

Comment:
These constants are repeated in a couple places. Should they get put somewhere and canonicalized then used across the app?